lib/check-or-update-webhooks.ts

```diff
@@ -10,24 +10,77 @@ import {
   getSections,
   toWebhook,
 } from ".";
-import currentWebhooks from "../payload-examples/index.json";
 
 const isNotNull = <T>(value: T | null): value is T => value !== null;
 
 export const checkOrUpdateWebhooks = async ({
   cached,
   checkOnly,
+  ghe,
+  githubAE,
+  updateAll,
 }: State): Promise<void> => {
-  const html = await getHtml({ cached });
+  if (updateAll) {
+    await checkOrUpdateWebhooks({
+      cached,
+      checkOnly,
+    });
+    await checkOrUpdateWebhooks({
+      cached,
+      checkOnly,
+      ghe: "",
+    });
+    await checkOrUpdateWebhooks({
+      cached,
+      checkOnly,
+      githubAE: true,
+    });
+  }
+
+  const [baseUrl, folderName] = ghe
+    ? [
+        `https://docs.github.com/en/enterprise-server@${ghe}/developers/webhooks-and-events/webhook-events-and-payloads`,
+        `ghes-${ghe.replace(".", "")}`,
+      ]
+    : githubAE
+    ? [
+        "https://docs.github.com/en/github-ae@latest/developers/webhooks-and-events/webhook-events-and-payloads",
+        "github.ae",
+      ]
+    : [
+        "https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/webhook-events-and-payloads",
+        "api.github.com",
+      ];
+
+  if (ghe === "") {
+    const gheVersions = ["2.19", "2.20", "2.21", "2.22", "3.0"];
+
+    for (let gheVersion of gheVersions) {
+      await checkOrUpdateWebhooks({
+        cached,
+        checkOnly,
+        ghe: gheVersion,
+      });
+    }
+
+    return;
+  }
```

Comment:
I think I would move the `ghe === ""` check above setting `baseUrl` and `folderName`. I was confused for a second how this works because of the order

Reply:
Done.